hedera-node/hedera-app/src/main/java/com/hedera/node/app/throttle/ThrottleAccumulator.java

```diff
@@ -150,12 +151,14 @@ public ThrottleAccumulator(
             @NonNull final ThrottleType throttleType,
             @NonNull final ThrottleMetrics throttleMetrics,
             @NonNull final LeakyBucketDeterministicThrottle gasThrottle,
-            @NonNull final LeakyBucketDeterministicThrottle bytesThrottle) {
+            @NonNull final LeakyBucketDeterministicThrottle bytesThrottle,
+            @NonNull final LeakyBucketDeterministicThrottle opsDurationThrottle) {
```

Comment:
`opsDurationThrottle` could easily be interpreted to apply to all types of ops

```suggestion
            @NonNull final LeakyBucketDeterministicThrottle contractOpsDurationThrottle) {
```

Reply:
updated.